tests/appsec/test_blocking_maxed.py

```diff
@@ -0,0 +1,84 @@
+from utils import context, released, coverage, interfaces, scenarios, weblog, bug, irrelevant
+
+
+@released(
+    cpp="?",
+    dotnet="2.30.0",
+    golang="1.48.0",
+    java="?",
```

Comment:
```suggestion
    java="1.7.0",
```

It should be complete for most variants on this release.

Reply:
But the following decorators should be added:
```
@missing_feature(context.weblog_variant == "spring-boot-native", reason="GraalVM. Tracing support only")
@missing_feature(context.weblog_variant == "spring-boot-3-native", reason="GraalVM. Tracing support only")
```

Reply:
done, I hope I put the decorators properly